api-reference/event-management/undo-ingested-events.mdx

```diff
@@ -0,0 +1,37 @@
+---
+openapi: delete /events
+---
+
+## Query Parameters: Allowed filter combinations
+<Warning>Every single event matching the filters will be deleted</Warning>
+<Info>Maximum events deleted per request is limited to 5. This behaviour will change in Event Correction API v2</Info>
+
+<AccordionGroup>
+<Accordion title="Parameters">
+`account_id` : Unique identifier of the account corresponding to the event
+
+`id` : Unique identifier of an event generated by Togai for reference
+
+`event_id`: Identifier of an event provided by the user
+
+`event_source_time`: Timestamp of an event provided by the user
+
+`created_at` : Timestamp corresponding to the event ingestion time
+</Accordion>
+</AccordionGroup>
+
+<AccordionGroup>
+<Accordion title="Allowed Combinations">
+`account_id`
+
+`account_id`, `id`
+
+`account_id`, `event_id`
+
+`account_id`, `event_source_time`
+
+`account_id`, `event_id`, `created_at`
+
+`account_id`, `event_id`, `event_source_time`
+</Accordion>
```

Comment:
Fine for now. We need to come up with a better way to represent this info

Reply:
Ok